Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -25,7 +25,7 @@ STPPaymentCardTextFieldDelegate, MessageBannerViewControllerPresenting {
   @IBOutlet private weak var zipcodeView: SettingsFormFieldView!
 
   private let supportedCardBrands: [STPCardBrand] = [.visa, .masterCard, .amex, .dinersClub,
-                                                     .discover, .JCB]
+                                                     .discover, .JCB, .unionPay]
```

Comment:
Could we order these alphabetically and maybe put each on a separate line for better readability?